android/src/main/java/com/reactnativecompressor/CompressorModule.java

```diff
@@ -119,4 +123,32 @@ public void getRealPath(String path,String type, Promise promise) {
       promise.reject(e);
     }
   }
+
+  @ReactMethod
+  public void getVideoMetaData(String filePath, Promise promise) {
+    try {
+      filePath=Utils.getRealPath(filePath,reactContext);
+      Uri uri= Uri.parse(filePath);
+      String srcPath = uri.getPath();
+      MediaMetadataRetriever metaRetriever = new MediaMetadataRetriever();
+      metaRetriever.setDataSource(srcPath);
+      File file=new File(srcPath);
+      float sizeInKBs = file.length()/1024;
```

Comment:
FYI this leads to a discrepancy between Android and iOS. On iOS, the "size" field is represented in bytes. On Android, the "size" field is converted to kilobytes. I'm happy to make a PR to address this, but how do you recommend we solve? This could result in breaking changes.

Reply:
@jeffaburt thanks for rising this issue. you can make PR for it. i will be happy to merge your PR